airbyte-webapp/src/pages/AdminPage/AdminPage.tsx

```diff
@@ -24,18 +26,21 @@ enum StepsTypes {
 }
 
 const AdminPage: React.FC = () => {
-  const steps = [
+  const steps: StepMenuItem[] = [
     {
       id: StepsTypes.SOURCES,
       name: <FormattedMessage id="admin.sources" />,
+      headTitle: <HeadTitle titleId="admin.sourcesHeadTitle" />,
```

Comment:
I do not think that title should be part of `StepMenuItem` and rendered anywhere in `StepMenu` because it is responsible for switching views only. 

Isn't it better to add title in Views ( SourcesView, DestinationsView, ConfigurationView )? What do you think?

Reply:
Yes, moved to views.